elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base), // needs to be #f0f0f0,
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
+  BorderRadius: pfe-var(ui--border-radius),
+  BorderWidth: pfe-var(ui--border-width),
```

Comment:
Just wondering if you need a local variable for border width? or could the properties below just lean on `pfe-var(ui--border-width)`